source/stylesheets/bootstrap/_navbar.scss

```diff
@@ -0,0 +1,87 @@
+.navbar.navbar-msm {
+  background-color: $msm-blue;
+  color: #FFF;
+  height: 60px;
+  border: 0;
+
+  a {
+    color: $msm-white;
+  }
+
+  .navbar-brand {
+    height: 60px;
+    font-size: 24px;
+    padding: 18px 20px;
+  }
+
+  .navbar-toggle {
+    border: 1px solid rgba(255,255,255,0.5);
+    margin: 13px 20px;
+
+    .icon-bar {
+      background-color: rgba(255,255,255,0.5);
+    }
+
+    &:hover, &:focus {
+      border-color: #FFF;
+      .icon-bar {
+        background-color: white;
+      }
+    }
+  }
+
+  ul.nav {
+    li > a {
```

Comment:
Please use direct siblings selector for list items. `ul > li`. In case you have nested lists in the future.